src/iconvert/CMakeLists.txt

```diff
@@ -2,4 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 # https://github.com/AcademySoftwareFoundation/OpenImageIO
 
-fancy_add_executable (LINK_LIBRARIES OpenImageIO)
+# Don't build if installing as a Python wheel
+if (SKBUILD)
+    return ()
+endif ()
+
```

Comment:
I think this idiom is not needed. The `fancy_add_executable` honors a variable called `ENABLE_binaryname` -- if set and is not on, that executable will not build. So instead of this return happening in each one, I think that in one central place where you set up SKBUILD, you can simply 

    set (ENABLE_iconvert OFF)
    set (ENABLE_idiff OFF)
    ... etc